### PR TITLE
fixed 2d visualizer so there is a dropdown choice

### PR DESCRIPTION
**Pull request by du-lab**
Hi Tomas,

Per your suggestion here https://github.com/mzmine/mzmine2/issues/284, there is now a dropdown choice to select either "Resample data" or "Use Raw data points" in the 2D visualizer.

Please let me know if you have any questions for me.

All the best,
Xiuxia on behalf of the Du-Lab Team

### CODE REVIEW

File: src/main/java/net/sf/mzmine/modules/visualization/twod/help/help.html~HEAD_0 [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<html>
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/visualization/twod/help/help.html~HEAD_0]: a bogus file here?
(also this one: help.html~HEAD)

File: src/main/java/net/sf/mzmine/modules/visualization/twod/PlotType.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package net.sf.mzmine.modules.visualization.twod;
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/visualization/twod/PlotType.java]: Please add the license header to all new files

**Comment by tomas-pluskal** [comment]
There seems to be a compilation error - see the Travis log details above

**Comment by du-lab** [comment]
could you check again?

**Comment by tomas-pluskal** [comment]
It compiles fine now. Could you also remove the bogus files I mentioned above? Thanks.

**Comment by tomas-pluskal** [comment]
Looks good, thanks!

**Pull request merged**